erlang/atomvm/Portfile

```diff
@@ -9,8 +9,8 @@ PortGroup               legacysupport 1.1
 # clock_gettime
 legacysupport.newest_darwin_requires_legacy 16
 
-github.setup            atomvm atomvm 0.6.0 v
-revision                1
+github.setup            atomvm atomvm 0.6.1 v
+revision                0
 epoch                   1
 categories              erlang devel
 maintainers             {pguyot @pguyot} openmaintainer
```

Comment:
your e-mail address is not correct, they way you have it now implies a ...@macports.org address. You should add the (obfuscated) domain as shown in the guide.

Reply:
@reneeotten Thank you for your review. I really mean to use my macports.org address for this, and I checked, it still works.

Reply:
oh, interesting... sorry, I see now that you indeed have a ...@macports.org e-mail address. You're listed though under retired developers/maintainers on the Wiki, that's probably why the bot doesn't label your PR as "Member" and why you don't have commit access.